snapcraft.yaml

```diff
@@ -236,15 +226,11 @@ parts:
       - po4a
       - po-debconf
       - zlib1g-dev
-      - libapparmor-dev
-      - libsnapd-glib-dev
```

Comment:
Do not remove libapparmor and libsnapd-glib. CUPS needs these libraries for Snap mediation, determining whether an incoming administrative request (create/modify queues, delete jobs, ...) comes from a confined Snap and whether the Snap plugs "cups-control" or not or whether it comes from an unsnapped app. This is an essential security feature for snapping apps, important part of the implementation of the "cups" and "cups-control" interfaces.

Reply:
This I need to check about a bit more. Do you have any doc about this?

Reply:
I am not sure how apparmor library is needed. I cannot see any of `cups-proxyd.c` or `proxy.c` linking to any of those libraries. But even then, the apparmor libraries are already in the core base snap.

```
/snap/core22/current/usr/lib/x86_64-linux-gnu/libapparmor.so.1
/snap/core22/current/usr/lib/x86_64-linux-gnu/libapparmor.so.1.8.2
```

Reply:
Okay I have found it out, where is snapd-glib linking